src/classes/Recipe.js

```diff
@@ -1,3 +1,4 @@
+const { testIngredients, recipeData, ingredientsData } = require("../data/testData")
```

Comment:
Ultimately, I don't think we're supposed to be using the testing dataset for anything besides testing. It looks like we aren't using `testIngredients` or `recipeData` anywhere, and `ingredientsData` is already being passed into the constructor. Was there a reason for importing the testing dataset here?

Reply:
Got it! I added the last two when I was testing this but I can definitely remove them. Should I remove the whole line?

Reply:
Yep! I'm pretty sure we're safe to remove the whole line.